src/types/PageRouters.ts

```diff
@@ -0,0 +1,5 @@
+export enum PageRouters {
+  home = '/',
+  people = '/people',
+  notFound = '*',
```

Comment:
```suggestion
export enum PageRouters {
  Home = '/',
  People = '/people',
  NotFound = '*',
```
Enum properties should be in PascalCase